keras_preprocessing/image.py

```diff
@@ -441,16 +444,21 @@ def save_img(path,
         **kwargs: Additional keyword arguments passed to `PIL.Image.save()`.
     """
     img = array_to_img(x, data_format=data_format, scale=scale)
+    if img.mode == 'RGBA' and (file_format == 'jpg' or file_format == 'jpeg'):
+        warnings.warn('The JPG format does not support '
+                      'RGBA images, converting to RGB.')
+        img = img.convert('RGB')
     img.save(path, format=file_format, **kwargs)
 
 
-def load_img(path, grayscale=False, target_size=None,
+def load_img(path, color_mode='rgb', target_size=None,
```

Comment:
Didn't see that the first time sorry. Butm this is a breaking change, please keep grayscale and deprecate it.